packages/hydrogen/src/analytics-manager/AnalyticsProvider.productView.example.tsx

```diff
@@ -0,0 +1,46 @@
+import {useLoaderData} from '@remix-run/react';
+import {json} from '@shopify/remix-oxygen';
+import {UNSTABLE_Analytics} from '@shopify/hydrogen';
+
+export async function loader() {
+  return json({
+    product: {
+      id: '123',
+      title: 'ABC',
+      vendor: 'abc',
+      selectedVariant: {
+        id: '456',
+        title: 'DEF',
+        price: {
+          amount: '100',
+        },
+      },
+    },
+  });
+}
+
+export default function Product() {
+  const {product} = useLoaderData<typeof loader>();
+  const {selectedVariant} = product;
+
+  return (
+    <div className="product">
+      <h1>{product.title}</h1>
+      <UNSTABLE_Analytics.ProductView
+        data={{
+          products: [
+            {
+              id: product.id,
+              title: product.title,
+              price: selectedVariant?.price.amount || '0',
```

Comment:
Is the right thing to do to `|| 0`? Or should it be `selectedVariant?.price.amount!`?